applications/GeoMechanicsApplication/python_scripts/geomechanics_solver.py

```diff
@@ -542,3 +558,13 @@ def _ConstructSolver(self, builder_and_solver, strategy_type):
             raise RuntimeError(f"Undefined strategy type '{strategy_type}'")
 
         return solving_strategy
+
+    def _GetResidualCriterion(self):
+        r_rt = self.settings["residual_relative_tolerance"].GetDouble()
+        r_at = self.settings["residual_absolute_tolerance"].GetDouble()
+        echo_level = self.settings["echo_level"].GetInt()
+        residual_criterion = KratosMultiphysics.ResidualCriteria(r_rt, r_at)
+        residual_criterion.SetEchoLevel(echo_level)
```

Comment:
I believe we can eliminate a temporary without sacrificing readability:
```suggestion
        residual_criterion.SetEchoLevel(self.settings["echo_level"].GetInt())
```

Reply:
Agreed, changed.